### PR TITLE
Update External Relations & Marketing Team Pages

### PR DESCRIPTION
**Pull request by leoyin1127**
Add pictures and member information in member.js

### CODE REVIEW

**Comment by HACKERALERT** [comment]
I think you should revert package-lock.json, it shouldn't be changed.

File: src/Assets/Lists/members.js [hunk 1]
```diff
@@ -108,25 +108,101 @@ const memberData = {
...
+                },
+                {
+                    fullName: "Dylan Tanaya Alianto",
+                    position: "Associate",
+                    emailLink: "dylan.alianto@mail.utoronto.ca",
+                    LinkedInLink: "https://www.linkedin.com/in/dylan-tanaya-alianto-7806a6229/",
+                },
+                {
+                    fullName: "Lauren Kang",
+                    position: "Director",
```
Comment (HACKERALERT) [inline src/Assets/Lists/members.js:178]: should director be moved up to below the other director?

**Comment by HACKERALERT** [comment]
Compress images to < 1MB each. Either size dowon or use tinyjpg.

File: src/Assets/Lists/members.js [hunk 2]
```diff
@@ -94,7 +94,7 @@ const memberData = {
                 {
                     fullName: "Zoey Xie",
                     position: "Logistics Associate",
-                    emailLink: "zoey.xie@mail.utoronto.ca",
+                    emaileLink: "zoey.xie@mail.utoronto.ca",
```
Comment (li-ruihao) [inline src/Assets/Lists/members.js:97]: Please fix this typo
```suggestion
                    emailLink: "zoey.xie@mail.utoronto.ca",
```

File: package-lock.json
Comment (li-ruihao) [inline package-lock.json:1]: Please do not commit this file

File: yarn.lock
Comment (li-ruihao) [inline yarn.lock:1]: Please do not commit this file

**Comment by leoyin1127** [comment]
all comments have been resolved; please review again if available, thanks

File: src/Assets/Lists/members.js [hunk 3]
```diff
@@ -108,25 +108,103 @@ const memberData = {
         },
         "External Relations": {
             membersList: [
-                
+                {
+                    fullName: "Archana Rajendran",
```
Comment (li-ruihao) [inline src/Assets/Lists/members.js:112]:
```suggestion
                    fullName: "Archana Reajendran",
```

File: src/Assets/Lists/members.js [hunk 4]
```diff
@@ -108,25 +108,103 @@ const memberData = {
...
+                    position: "Director",
+                    emailLink: "a.rajendran@mail.utoronto.ca",
+                },
+                {
+                    fullName: "Chu Qiao",
```
Comment (li-ruihao) [inline src/Assets/Lists/members.js:117]:
```suggestion
                    fullName: "Jocelyn Qiao",
```

File: src/Assets/Lists/members.js [hunk 5]
```diff
@@ -108,25 +108,103 @@ const memberData = {
         },
         "External Relations": {
             membersList: [
-                
+                {
+                    fullName: "Archana Rajendran",
+                    position: "Director",
```
Comment (li-ruihao) [inline src/Assets/Lists/members.js:113]:
```suggestion
                    position: "External Relations Director",
```

File: src/Assets/Lists/members.js [hunk 6]
```diff
@@ -108,25 +108,103 @@ const memberData = {
...
+                    emailLink: "a.rajendran@mail.utoronto.ca",
+                },
+                {
+                    fullName: "Chu Qiao",
+                    position: "Director",
```
Comment (li-ruihao) [inline src/Assets/Lists/members.js:118]:
```suggestion
                    position: "External Relations Director",
```

File: src/Assets/Lists/members.js [hunk 7]
```diff
@@ -108,25 +108,103 @@ const memberData = {
...
+                    emailLink: "chu.qiao@mail.utoronto.ca",
+                    LinkedInLink: "https://www.linkedin.com/in/jocelynqiaochu/",
+                },
+                {
+                    fullName: "Dooyeon Yeom",
```
Comment (li-ruihao) [inline src/Assets/Lists/members.js:123]:
```suggestion
                    fullName: "Dunny Yeom",
```

File: src/Assets/Lists/members.js [hunk 8]
```diff
@@ -108,25 +108,103 @@ const memberData = {
...
+                    LinkedInLink: "https://www.linkedin.com/in/royce-luo-63b801244/",
+                },
+                {
+                    fullName: "Tanvi Bansal",
+                    position: "Associate",
+                    emailLink: "tanvi.bansal@mail.utoronto.ca",
+                    LinkedInLink: "https://in.linkedin.com/in/tanvi--bansal",
+                },
+                {
+                    fullName: "Victoria Chen",
```
Comment (li-ruihao) [inline src/Assets/Lists/members.js:147]: Is she in the team??? How come I don't see her anywhere whether it is in org chart or anything? Let me confirm

File: src/Assets/Lists/members.js [hunk 9]
```diff
@@ -108,25 +108,103 @@ const memberData = {
...
             ],
+
             firstRowSize: 2,
         },
         "Marketing": {
             membersList: [
-                
+                {
+                    fullName: "Arnav Patil",
+                    position: "Director",
```
Comment (li-ruihao) [inline src/Assets/Lists/members.js:160]:
```suggestion
                    position: "Marketing Director",
```

File: src/Assets/Lists/members.js [hunk 10]
```diff
@@ -108,25 +108,103 @@ const memberData = {
...
+                    emailLink: "arnav.patil@mail.utoronto.ca",
+                    LinkedInLink: "https://www.linkedin.com/in/arnavpatil/",
+
+                },
+                {
+                    fullName: "Lauren Kang",
+                    position: "Director",
```
Comment (li-ruihao) [inline src/Assets/Lists/members.js:167]:
```suggestion
                    position: "Marketing Director",
```

File: src/Assets/Lists/members.js [hunk 11]
```diff
@@ -108,25 +108,103 @@ const memberData = {
...
+                    emailLink: "yj.kang@mail.utoronto.ca",
+                    LinkedInLink: "https://www.linkedin.com/in/laurenyjkang/",
+                },
+                {
+                    fullName: "Daniel Rolfe",
+                    position: "Associate",
```
Comment (li-ruihao) [inline src/Assets/Lists/members.js:173]:
```suggestion
                    position: "Senior Associate",
```

File: src/Assets/Lists/members.js [hunk 12]
```diff
@@ -108,25 +108,103 @@ const memberData = {
...
+                {
+                    fullName: "Dylan Tanaya Alianto",
+                    position: "Associate",
+                    emailLink: "dylan.alianto@mail.utoronto.ca",
+                    LinkedInLink: "https://www.linkedin.com/in/dylan-tanaya-alianto-7806a6229/",
+                },
+
+                {
+                    fullName: "Mary Ghazi",
+                    position: "Senior Associate",
```
Comment (li-ruihao) [inline src/Assets/Lists/members.js:186]: Please put all senior associates ahead of associates, thanks.

File: src/Assets/Lists/members.js [hunk 13]
```diff
@@ -108,25 +108,103 @@ const memberData = {
...
+                    emailLink: "mary.ghazi@mail.utoronto.ca",
+                    LinkedInLink: "https://www.linkedin.com/in/mary-ghazi",
+                },
+                {
+                    fullName: "Yunxiang Xu",
```
Comment (li-ruihao) [inline src/Assets/Lists/members.js:191]:
```suggestion
                    fullName: "David Xu",
```

**Review by li-ruihao — CHANGES_REQUESTED** [review]
I've done the review for the first time, please make the changes asap, thanks

File: src/Assets/Lists/members.js (hunk 8 above)
Comment (li-ruihao) [reply src/Assets/Lists/members.js:147]: OK confirmed she is in the team. Also, there is a new ER member that needs to be added, her name is Melissa Cao and you can use her Slack Profile Picture, ignore linkedIn and email for now.

**Comment by leoyin1127** [comment]
i've finished all the changes required, please review again

**Review by li-ruihao — APPROVED** [review]

**Pull request merged**